packages/core/src/cli/commands/clear-cache.command.ts

```diff
@@ -20,21 +20,21 @@ const program = new DmnoCommand('clear-cache')
 
 program.action(async (opts, more) => {
```

Comment:
TODO: this is pseudo-broken at the moment, it always re-resolves before then deletes the file